src/lib/logging/auditLogs.ts

```diff
@@ -22,26 +23,7 @@ export enum AuditSubjectType {
   Response = "Response",
 }
 
-let queueUrlRef: string | null = null;
-
-const getQueueUrl = async () => {
-  if (!queueUrlRef) {
-    const data = await AwsServicesConnector.getInstance().sqsClient.send(
-      new GetQueueUrlCommand({
-        QueueName: "api_audit_log_queue",
-      }),
-    );
-    queueUrlRef = data.QueueUrl ?? null;
-    logMessage.debug(`Audit Log Queue URL initialized: ${queueUrlRef}`);
-  }
-  return queueUrlRef;
-};
-
-//Initialise the queueUrlRef on load except when running tests
-// This mock should be refactored when we once we have a better way to mock the AWS SDK
-process.env.NODE_ENV !== "test" && getQueueUrl();
-
-export const logEvent = async (
+export const publishAuditLog = (
```

Comment:
Any specific reason for renaming the function?  My thought was to keep the function name similar between the App and Api code bases.

Reply:
Will rename to `auditLog`